types/facebook-js-sdk/index.d.ts

```diff
@@ -7,6 +7,8 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
+declare function fbAsyncInit(): void;
```

Comment:
```suggestion
interface Window {
    fbAsyncInit?: () => void;
}
```
just for consideration: why not be explicit what it does? (sets handler directly on window, we just know global is the same scope, unless not in a browser)

Reply:
I just think `declare function fbAsyncInit` has a similar declaration style to `declare var FB`. What do you think?

Reply:
yes, agree. Just pointed their sdk declares FB symbol, while they do not declare `fbAsyncInit`, they are just consuming this, if exists